astroquery/query.py

```diff
@@ -303,14 +340,14 @@ def _request(self, method, url,
                 return local_filepath
         else:
             query = AstroQuery(method, url, **req_kwargs)
-            if ((self.cache_location is None) or (not self._cache_active) or (not cache)):
-                with suspend_cache(self):
+            if (self.cache_location is None) or (not cache):
```

Comment:
`self.cache_location` will default to the configuration item value, so it can't be `None` anymore:
```suggestion
            if not cache:
```